app/views/users/index.html.erb

```diff
@@ -4,7 +4,8 @@
       <li>
         Name: <%= user.name %>
         <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>&nbsp;
+          <%= link_to 'Add friend',  user_path(user), class: 'profile-link' %>
```

Comment:
I think is a mistake here.
`Add Friend` should not open the **user detail page**